### ISSUE TITLE
Open AccordionSections are closed when they are out of the screen

### ISSUE THREAD

**Issue by omerbengal** [GotJimmy/accordion#56]
I have a list of about 20 elements, for each one of them I have an AccordionSection with content inside.
Let's say I open (for example) the first and the last AccordionSection.
Now I am at the top of the list with the first AccordionSection opened, and when I scroll down quickly - I can see the last AccordionSection marked as open (with the color chosen in the parameter "headerBackgroundColorOpened") - but it is not really open - and by that I mean that I cant see the content. Only a second after the last AccordionSection is inside the screen the content is being opened and revealed (at the start it was not inside the screen - it got into the screen by scrolling).

![Simulator Screen Recording - iPhone 14 - 2023-10-05 at 23 13 05](https://github.com/GotJimmy/accordion/assets/39809022/1877a4e5-b2f6-4d22-a4c5-2b6117f8dff3)

To sum up - it seems like AccordionSections' content is closing when out of sight.

**Comment by GotJimmy** [comment]
Have you tried setting `maxOpenSections` to a higher number? I tried it with a value of 3 and it works just fine.

**Comment by omerbengal** [comment]
It was set to list.length.
But to make sure - I changed it to 100.
same result...

**Comment by GotJimmy** [comment]
Ok, it's not closing but only the sections on screen will be rendered. When going back to a section currently off screen it plays the animation again for this section.
